reference/conanfile/tools/gnu/pkgconfigdeps.rst

```diff
@@ -181,6 +181,29 @@ Their ``pkg_config_name`` properties are used as the final PC file names:
     Requires: zlib
 
 
+A special mention when a component shares the same ``*.pc`` file name as the root package one:
+
+.. code:: python
+
+    from conans import ConanFile
+
+    class OpenCLConan(ConanFile):
+
+        # ...
+
+        def package_info(self):
+            self.cpp_info.set_property("pkg_config_name", "OpenCL")  # -> OpenCL.pc
+            self.cpp_info.components["_opencl-headers"].set_property("pkg_config_name", "OpenCL")  # -> OpenCL.pc
+
+The only ``*.pc`` file created will be the one belonging to the component:
+
+- OpenCL.pc (from component)
+
+.. important::
+
+    Components ``*.pc`` file names have more priority than package one in case of collision.
```

Comment:
I don't know if this important note is worth it...

Reply:
I agree. I was doubting if putting it or not. I guess it's enough with the explanation before.